calculator.cc

```diff
@@ -6,17 +6,33 @@
 
 namespace dev {
 
+bool Calculator::isValidExpression(std::vector<std::string> tokens) {
+    int noOfOperands = 0;
+    for (auto token : tokens) {
+        if ((noOfOperands < 2) && (!Operand::isOperand(token))) {
```

Comment:
auto enouh_operands for operator = []() {
}

Reply:
Done.